docs/Open Data Model/Open Data Model.md

```diff
@@ -0,0 +1,755 @@
+|name_last|string|Last name|Smith
+|name_mgr|string|Manager’s name|Ronald Reagan
+|phone|string|Phone number|703-555-1212
+|email|string|Email address|jsmith@company.com
+|code|string|Job code|3455
+|loc|string|Location|US
+|departm|string|Department|IT
+|dn||Distinguished name|"CN=scm-admin-mej-test2-adk,OU=app-|admins,DC=ad,DC=halxg,DC=companya,DC=com"
+|ou|string|Organizational unit|EAST
+|empid|string|Employee ID|12345
+|title|string|Job Title|Director of IT
+|groups|string (comma separated list, no spaces after comma)|Groups to which the user belongs|“Domain Admins”, “Domain Users”
+|dvc_type|string|Device type that generated the user context data|Active Directory
+|dvc_vendor|string|Vendor|Microsoft
+|dvc_version|string|Version |8.1.2
+|additional_attrs|string|Additional attributes of user|Key value pairs
+
+
+**Endpoint Context Model**
+------------------
+The data model for endpoint context information is as follows:
+|**Abbreviation**|**Data Type**|**Description**|**Sample Values**|
+|---|---|---|---|
+|dvc_time|bigint|Timestamp from when the endpoint context information is obtained|1472653952
+|ip4|bigint|IP address of endpoint |Integer representaion of 10.1.1.1
+|ip6|bigint|IP address of endpoint |Integer representaion of 10.1.1.1
+|os|string|Operating system|Redhat Linux 6.5.1
+|os_version|string|Version of OS|5.4
+|os_sp|string|Service pack|SP 2.3.4.55
+|tz|string|timezone|EST
+|hotfixes|string|Applied hotfixes|993.2
+|disks|string|Available disks|\\Device\\HarddiskVolume1, \\Device\\HarddiskVolume2
+|removables|string|Removable media devices|USB Key
+|nics|string|Network interfaces|fe10::28f4:1a47:658b:d6e8, fe82::28f4:1a47:658b:d6e8 
+|drivers|string|Installed kernel drivers|ntoskrnl.exe, hal.dll
+|users|string|Local user accounts|administrator, jsmith
+|host|string|Hostname of endpoint|tes1.companya.com
+|mac|string|MAC address of endpoint|fe10::28f4:1a47:658b:d6e8
+|owner|string|Endpoint owner (name)|John Smith
+|vulns|string (comma separated, no spaces after commas)|Vulnerability identifiers (CVE identifier)|CVE-123, CVE-456
+|loc|string|Location|US
+|departm|string|Department name|IT
+|company|string|Company name|CompanyA
+|regs|string (comma-separated)|Applicable regulations|HIPAA, SOX
+|svcs|string (comma-separated)|Services running on system|Cisco Systems, Inc. VPN Service, Adobe LM Service
+|procs|string|Processes|svchost.exe, sppsvc.exe
+|criticality|string|Criticality of device|Very High
+|apps|string (comma-separated)|Applications running on system|Microsoft Word, Chrome
+|desc|string|Endpoint descriptor|Some string
+|dvc_type|string|Device type that generated the log|Microsoft Windows 7
+|dvc_vendor|string|Vendor|Endgame
+|dvc_version|string|Version |2.1
+|architecture|string|CPU architecture|x86
+|uuid|string|Universally unique identifier|a59ba71e-18b0-f762-2f02-0deaf95076c6
+|memtotal|int|Total memory (bytes)|844564433
+|additional_attrs|string|Additional attributes|Key value pairs
+
+**VPN Context Model**
```

Comment:
Out of curiosity, where is this data model coming from? I haven't seen this before...